lib/dynamic_config/dcdo.rb

```diff
@@ -45,6 +45,8 @@ def frontend_config
       # Whether the scholarship dropdown is locked on the application dashboard.
       'scholarship-dropdown-locked': DCDO.get('scholarship-dropdown-locked', true),
       hoc_mode: DCDO.get('hoc_mode', false),
+      # Whether to show the marketing banners for the AI Teacher Assistant launch. Can be removed later.
+      'ai-teaching-assistant-launch': DCDO.get('ai-teaching-assistant-launch', false),
```

Comment:
Just checking -- do we need to reference this in JS?

Reply:
I don't think so 🤔 Where were you thinking this would go?

Reply:
You only need to add DCDO flags to this file if you want to access them in javascript. If you're only accessing them in haml/ruby code, you can just include it in the PR.

Reply:
(I'm like 99% sure but now I'm wondering if I missed a memo lol)

Reply:
Oh! I've always added them here for use across all sites.

Reply:
I don't think it's needed if you're only using it from the server-side, but it's safe to add here either way. Up to you :)